src/surface/surface.scss

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-surface {
```

Comment:
Are you sure about this name? Surface is too generic, and use case is only type of surface

Reply:
We threw around some name ideas in the layouts call, I liked "surface" most.  Greta pointed out MUI uses "paper".  I'm open to whatever we want to call it.

Reply:
ok but you're missing the important part: **_this use case is only <u>one</u> type of surface_**

What if the user wants a background-2 or background-3 surface? What if they don't want the shadow or want different elevation? All of those things should be configurable in a `<Surface>` component imo.

Reply:
I've added support for different elevations.  I cant think of an instance where changing background color would be needed, we can add that later if we need it.

Reply:
For me "surface" also sounds too generic. Also, we are kind of copying this from Material UI which has category named Surface that contains these kind of components. As I told in layouts meeting, I would go for the "Card".

Reply:
Card can be confused with Tile

Reply:
I don't think `Card` is the best option as most people think that card should have more elements than just background and shadow. 
![image](https://user-images.githubusercontent.com/83585998/159008839-d50250fe-3125-4c81-8a7e-ec85fd30de32.png)

Maybe `iui-elevated-background`? or `iui-elevated-surface`?

Reply:
We could add semi-transparent or colored surfaces later

Reply:
Surface makes the most sense to me.  It gives you an elevated "surface" above bg2 to place content, whether it be a tile looking thing or a dropdown menu.
![Screen Shot 2022-03-18 at 9 29 34 AM](https://user-images.githubusercontent.com/849817/159011573-49bcfc85-59f1-45f2-b745-d41934ee9ae0.png)

Reply:
I like `Card` :D